src/cli/cli.cpp

```diff
@@ -7212,13 +7233,34 @@ template <> otError Interpreter::Process<Cmd("uptime")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli uptime
+     * @code
+     * uptime
+     * 12:46:35:469
```

Comment:
Replace last `:` with `.`:
```suggestion
     * 12:46:35.469
```

Reply:
done, thx for catching this.